src/utils/toJson.tsx

```diff
@@ -1,4 +1,14 @@
 const toJson = (type: string, jsonld: any) => {
+  const jsonLdObject = Array.isArray(jsonld)
```

Comment:
I wonder would a third param of `isArray` be a bit more resilient? 

I know there are examples of this already in the codebase though and it works well. 

Happy to leave as is.